src/Features/Core/Portable/Shared/Extensions/DocumentExtensions.cs

```diff
@@ -14,7 +14,8 @@ internal static class DocumentExtensions
     {
         public static bool ShouldHideAdvancedMembers(this Document document)
         {
-            return document.Options.GetOption(CompletionOptions.HideAdvancedMembers);
+            // Since we don't actually have a way to configure this per-document, we can fetch from the core workspace
+            return document.Project.Solution.Workspace.Options.GetOption(CompletionOptions.HideAdvancedMembers, document.Project.Language);
```

Comment:
My basic understanding was that editorconfig was for things impacting the actual code, not the ide experience (e.g. this option that just effects the completion list experience). This matches with the list of [generally-adopted editorconfig properties](https://github.com/editorconfig/editorconfig/wiki/EditorConfig-Properties). But maybe Jason had other reasons.